README.md

```diff
@@ -52,6 +52,12 @@ We are always looking for new contributors, if you're willing to help please let
 - [Project Smoothie Landing page](https://github.com/openfoodfacts/smoothielanding)
 - Private app signing for iOS certificates repository - please ask @teolemon
 
+## Forks we depend on
+- [g123k/google_ml_barcode_scanner](https://github.com/g123k/google_ml_barcode_scanner) - Because XYZ
+- [g123k/plugins](https://github.com/g123k/plugins) - Because XYZ - folder: ABC, DEF
+- [g123k/realm-dart_ios_only](https://github.com/g123k/realm-dart_ios_only) - Because XYZ
+- [g123k/sqflite_android_only](https://github.com/g123k/sqflite_android_only) - Because XYZ
+- [M123-dev/matomo-tracker](https://github.com/M123-dev/matomo-tracker) - Because XYZ
```

Comment:
```suggestion
- [g123k/plugins](https://github.com/g123k/plugins) - Because XYZ - folder: ABC, DEF
- [g123k/realm-dart_ios_only](https://github.com/g123k/realm-dart_ios_only) - Because XYZ
- [g123k/sqflite_android_only](https://github.com/g123k/sqflite_android_only) - Because XYZ
- [M123-dev/matomo-tracker](https://github.com/M123-dev/matomo-tracker) - TODO: switch to vanilla
```